docs/src/main/paradox/connectors.md

```diff
@@ -0,0 +1,59 @@
+# Connectors
```

Comment:
This file was removed in master. You should add a link to `couchbase.md` in in `index.md`, replacing the reference to the external library and list that in the documentation instead.

As used on the [MongoDb page](https://developer.lightbend.com/docs/alpakka/current/mongodb.html).